tf2_eigen/CMakeLists.txt

```diff
@@ -14,11 +14,20 @@ find_package(geometry_msgs REQUIRED)
 find_package(tf2 REQUIRED)
 find_package(tf2_ros REQUIRED)
 
-find_package(eigen3_cmake_module REQUIRED)
 find_package(Eigen3 REQUIRED)
 
-install(DIRECTORY include/${PROJECT_NAME}/
-        DESTINATION include/${PROJECT_NAME})
+add_library(tf2_eigen INTERFACE)
+target_link_libraries(tf2_eigen INTERFACE
+  Eigen3::Eigen
+  tf2::tf2
+  tf2_ros::tf2_ros)
+target_include_directories(tf2_eigen INTERFACE
+  "$<BUILD_INTERFACE:${CMAKE_CURRENT_SOURCE_DIR}/include>"
+  "$<INSTALL_INTERFACE:include/${PROJECT_NAME}>")
+
+
```

Comment:
Nit: extra newline.

Reply:
Removed extra newline in https://github.com/ros2/geometry2/pull/493/commits/182554734478d9fa8904ca45dfb78a90c836ea06